src/components/pages/order/Main/Panel/PanelTabs.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { useContext } from "react";
+import PanelContext from "../../../../context/OrderContext";
+import { FiChevronDown, FiChevronUp } from "react-icons/fi";
+import styled from "styled-components/macro";
+import Tab from "../../../../reusable-ui/Tab";
+import { theme } from "./../../../../../theme/index";
+import { configTabs } from "./configTabs";
+
+function PanelTabs() {
+  //// revoir les condition true et false de tabicon et add ** edit en position collapsed avec icon
+  const {
+    tabSelected,
+    setTabSelected,
+    tabIconSelected,
+    setTabIconSelected,
+    isCollasped,
+    setIsCollasped,
+  } = useContext(PanelContext);
+
+  const isTabSelected = (tab) => {
```

Comment:
```suggestion
  const selectTab = (tab) => {
```

ce n'est pas un booléan alors pourquoi l'appeler isTabSelected ?